csrankings-r.csv

```diff
@@ -1,5 +1,6 @@
 name,affiliation,homepage,scholarid
 R. A. Bodnar,University of Southampton,https://www.ecs.soton.ac.uk/people/rab1e14,NOSCHOLARPAGE
+R. B. Fisher,University of Edinburgh,https://homepages.inf.ed.ac.uk/rbf/,LigYduEAAAAJ

```

Comment:
Names must correspond to DBLP names - these appear to be "Robert B. Fisher" and "Bob Fisher". Both need to be included, and "R. B. Fisher" should be removed.